docs/content/advanced/backend/payment/how-to-create-payment-provider.md

```diff
@@ -53,51 +53,44 @@ For example, create the file `src/services/my-payment.ts` with the following con
 <!-- eslint-disable max-len -->
 
 ```ts title=src/services/my-payment.ts
-import { 
-  AbstractPaymentService, 
-  Cart, Data, Payment, PaymentSession, 
-  PaymentSessionStatus, TransactionBaseService, 
-} from "@medusajs/medusa"
+import { AbstractPaymentService, Context, Data, Payment, PaymentSession, PaymentSessionStatus, TransactionBaseService } from "@medusajs/medusa"
 import { EntityManager } from "typeorm"
 
 class MyPaymentService extends AbstractPaymentService<TransactionBaseService> {
   protected manager_: EntityManager
   protected transactionManager_: EntityManager
 
-  getPaymentData(paymentSession: PaymentSession): Promise<Data> {
+  async getPaymentData(paymentSession: PaymentSession): Promise<Data> {
     throw new Error("Method not implemented.")
   }
-  updatePaymentData(paymentSessionData: Data, data: Data): Promise<Data> {
+  async updatePaymentData(paymentSessionData: Data, data: Data): Promise<Data> {
     throw new Error("Method not implemented.")
   }
-  createPayment(cart: Cart): Promise<Data> {
+  async createPayment(context: Context): Promise<PaymentSessionResponse> {
```

Comment:
todo: `Cart & PaymentContext)`

Reply:
The thing is that TS will error out because the typings are mismatched.

Reply:
Ah ok I didn't notice that. Will make the change!